src/proofs/ProofSteps.h

```diff
@@ -106,8 +106,8 @@ class StepHandler {
                 Normalizer::Equalities const & normalizingEqualities, std::ostream & out, Logic & logic,
                 ChcDirectedHyperGraph originalGraph)
         : derivation(std::move(derivation)), originalAssertions(std::move(originalAssertions)),
-          normalizingEqualities(std::move(normalizingEqualities)), originalGraph(std::move(originalGraph)), out(out),
-          logic(logic) {}
+          normalizingEqualities(normalizingEqualities), originalGraph(std::move(originalGraph)), out(out),
+          logic(logic), currStep(0), renamingTransIndex(0), renamingCongIndex(0) {}
```

Comment:
I actually preferred the previous version, where the members were initialized inline.
And what about the `transitivityStep`?